lagotto/init.sls

```diff
@@ -87,10 +103,12 @@ lagotto-sysctl-ip-local-port-range:
     - name: net.ipv4.ip_local_port_range
     - value: {{ ip_local_port_range }}
 
+{%- if oscodename == 'trusty' %}
 lagotto-sysctl-tcp-tw-recycle:
   sysctl.present:
     - name: net.ipv4.tcp_tw_recycle
     - value: {{ tcp_tw_recycle }}
+{%- endif %}
```

Comment:
I'm not sure that we don't need to do this for 18.04. This was an issue with exhausting ephemeral tcp ports when processing an ALM source (PMC). This may not be an issue with a newer version of CouchDB (ie, 1.5 -> 2.x). dunno. needs a little research.

Reply:
We'll need to activate it another way, it is missing from `sysctl` in 1804, I'll dig more into it to see if we can keep it consistent, otherwise we'll need to update the `/etc/sysctl.conf`

Reply:
It does seem, for whatever reason, this would need to be set in the conf file instead now. Having done some reading most opinions are that this should be disabled, especially behind a NAT, so I'd like to try keeping it disabled on the new hosts and seeing if all goes well.